### DISCUSSION TITLE
Combat statistics

### DISCUSSION THREAD

**Discussion by xTattered** [TI4-Online/TI4-TTPG#371] [General]
Does your extension log combat statistics? 

for instance ship type, who fought, who won?

**Comment by darrellanderson** [comment]
No, it only tracks the current combat state.  How would you use that information?

**Reply by xTattered** [reply]
I have a theory that there is an inverse correlation between number of combats a player participates in and win rate of that player regardless of faction. Or, if this isn't true, which factions need to fight to win and which factions should not fight? I was also curious about popular fleet compositions and amount/type of ships built over the course of the game and wondering if it was accessible in the dateset. 

I was able to find board states and unit upgrade frequency on the stats dashboard which is close, but doesn't scratch the itch entirely. I have found myself upgrading units only to find that I never roll dice with them in several of my games. 

I was curious about looking more into the combat meta of the game, that's all.

**Reply by darrellanderson** [reply]
It does have a snapshot of the map at the start of each round, including player units.  That would give you some insights into idle fleet compositions, but not necessarily what players bring to a combat.
